vagen/envs/navigation/scripts/create_tasks.py

```
@@ -0,0 +1,184 @@
+        gridSize=GRID_SIZE,
+        scene="FloorPlan1",
+        width=300,
+        height=300,
+        platform=CloudRendering,
+    )
+
+    for scene in ALL_SCENES:
+        try:
+            controller.reset(scene=scene)
+        except Exception:
+            continue
```

Ruff (0.15.6)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)